Neo4j.Driver/Neo4j.Driver/Experimental/ExperimentalExtensions.cs

```diff
@@ -34,4 +34,17 @@ public static SessionConfigBuilder WithBookmarkManager(this SessionConfigBuilder
     {
         return builder.WithBookmarkManager(bookmarkManager);
     }
+    
+    /// <summary>
+    /// There is no guarantee that anything in Neo4j.Driver.Experimental namespace will be in a next minor version.
+    /// Experimental: This method be removed, in favor of "BookmarkManager" property on <see cref="SessionConfig"/>.
```

Comment:
```suggestion
    /// Experimental: This method will be removed, in favor of "BookmarkManager" property on <see cref="SessionConfig"/>.
```
I think a verb is missing.